wtxmgr/tx.go

```diff
@@ -404,7 +404,9 @@ func (s *Store) addCredit(ns walletdb.ReadWriteBucket, rec *TxRecord, block *Blo
 		if existsRawUnminedCredit(ns, k) != nil {
 			return false, nil
 		}
-		if existsRawUnspent(ns, k) != nil {
+		if _, tv := latestTxRecord(ns, &rec.Hash); tv != nil {
+			log.Infof("addCredit for new transaction %v is ignored as it "+
```

Comment:
Nit: no need for this log. Or perhaps demote to `trace` and reword to `Ignoring credit for existing confirmed transaction %v`.

Reply:
Fixed.